lcx/src/adapter.ts

```diff
@@ -0,0 +1,33 @@
+import { Execute } from '@chainlink/types'
+import { Requester, Validator } from '@chainlink/external-adapter'
+
+const customError = (data: any) => {
+  return data.Response === 'Request failed with status code 400'
```

Comment:
I do not think this is necessary in this case. The Requestor will actually throw on error and this function will not be called.

This custom handler is for cases where API returns 200, but we don't get the expected data in the response so we force an error. I think we should rarely use it, as in most cases most APIs will return the correct code (non 200) for error cases, and our requester will throw an error.

Reply:
Good to know, thanks. Removed the `customError`.